hw2/03-count.js

```diff
@@ -1,3 +1,21 @@
-// Add your code here
+const wordToBeHighlighted = document.getElementById("word");
+const text = document.getElementById("text");
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeyDown() {
+  const searchWord = wordToBeHighlighted.value.trim().toLowerCase();
+  const textContent = text.textContent.toLowerCase();
```

Comment:
I think it would be best to not modify the casing of the original text. I would instead try to toLowerCase() trimmedword to preserve the text.

Reply:
I have made the changes as per your suggestion. Please let me know if this is correct or do I need to make any other changes.